### DISCUSSION TITLE
[Feature] skip scanning external directories by name

### DISCUSSION THREAD

**Discussion by Luvbe3** [immich-app/immich#5919] [Feature Request]
### The feature

i use synology photos and want to add synology photo upload folder to immch. But in each folder of synology photos there is a hidden folder named @eaDir. contains photo and video files created by synology. Scanning the entire @eaDir folder will duplicate photos and videos!
Maybe there's a feature to scan that skips folders by name which would be nice!

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by schuhbacca** [comment]
I think others have used exclusion patterns before?
https://immich.app/docs/features/libraries

**Comment by Luvbe3** [comment]
> @eadir

Thank you. let me try

**Comment by ydrob** [comment]
@Luvbe3 
You should use this pattern to exclude syno folders 
![image](https://github.com/immich-app/immich/assets/77321248/12fc4a78-23ba-4c83-9827-6d184785a45c)
On github it is impossible to use special symbols
It works for me like a charm.

**Comment by Gitonthegrid** [comment]
And how to exclude the #recycle ?

**Comment by ydrob** [comment]
![image](https://github.com/user-attachments/assets/d4704bda-5fd0-4df9-af30-d2a0cb2976e4)
The same

**Reply by zecka** [reply]
For copy past ;)
- `**/\@eadir/**`
- `**/\#recycle/**`

**Reply by juanbretti** [reply]
I should use `**/\@eaDir/**` or `**/@eaDir/**` as mentioned at https://github.com/immich-app/immich/discussions/7573#discussioncomment-8652735
I mean, should I escape the `@` symbol?

**Reply by ydrob** [reply]
Yes, you should.
`**/\@eadir/**`
